tests/test_skills/test_base.py

```diff
@@ -153,7 +153,7 @@ def setup_class(cls):
         cls._patch_logger()
 
         cls.cwd = os.getcwd()
-        cls.t = tempfile.mktemp()
+        cls.t = NamedTemporaryFile()
```

Comment:
Note: https://bandit.readthedocs.io/en/latest/blacklists/blacklist_calls.html#b306-mktemp-q

so `tempfile.mktemp()` needs to be replaced. Not sure my approach is correct.